tests/test_pulsar.py

```diff
@@ -185,25 +186,41 @@ def setUpClass(cls):
             timing_package="pint",
         )
 
-    # exclude tests pending implementation of .stoas, .dm, .dmx in PintPulsar
+    def test_dm(self):
+        """Check DM/DMX access."""
 
-    def test_stoas(self):
-        assert hasattr(self.psr, "stoas")
+        msg = "dm value incorrect"
+        assert self.psr.dm == np.longdouble("13.299393"), msg
 
-    def test_dm(self):
-        assert hasattr(self.psr, "dm")
+        msg = "dmx struct incorrect (spotcheck)"
+        assert len(self.psr.dmx) == 72, msg
+        assert self.psr.dmx["DMX_0001"]["DMX"] == np.float64("0.015161863"), msg
+        assert self.psr.dmx["DMX_0001"]["fit"], msg
 
-    def test_planetssb(self):
-        assert hasattr(self.psr, "planetssb")
 
-    def test_sunssb(self):
-        assert hasattr(self.psr, "sunssb")
+class TestPulsarPintRADEC(unittest.TestCase):
```

Comment:
Same story as test_deterministic_signals.py... can we avoid duplicating tests for PINT?

Also, I think

```
def test_deflate_inflate(self):
         pass
```

needs to stay... unless deflation actually works with PINT?

Reply:
Alright, I think these are taken care of. There are a few other Pint specific tests. The DM variable check is different because the values come out as `numpy.float64`, not `longdouble`. The RA DEC import checks the `AstrometryEquatorial` option for `pos_t` and the no planet tag checks an exception in the `PintPulsar` class.